src/modules/users/components/UserProfileEdit/UserProfileEdit.tsx

```diff
@@ -52,6 +54,14 @@ const MSG = defineMessages({
     id: 'users.UserProfileEdit.labelLocation',
     defaultMessage: 'Location',
   },
+  toastSuccess: {
+    id: 'users.UserProfileEdit.toastSuccess',
+    defaultMessage: 'Profile settings have been updated.',
+  },
+  toastError: {
+    id: 'users.UserProfileEdit.toastError',
+    defaultMessage: 'Profile settings were not able to be updated. Try again.',
+  },
```

Comment:
These can be combined using `select`.

Reply:
I prefer these individual msgs being kept separate as they are serving different outcomes.